ROADMAP.md

```diff
@@ -1,19 +1,30 @@
-# kompose roadmap
+# Kompose roadmap
 
-This document is a general overview of all upcoming development for `kompose`. The dates below are not considered set-in-stone, but rather an approximation of each release.
-
-We track each release by using GitHub [milestones](https://github.com/kubernetes-incubator/kompose/milestones).
+This document is a general overview of upcoming development for `kompose`.
+It lists only important features and general direction for for upcoming releases. 
+The dates below are not considered set-in-stone, but rather an approximation of each release. 
 
 ## Upcoming releases
-Kompose is released every last week of the month.
+Kompose is on time based release cycle. 
+Current target is release every last week of the month.
+
+### 0.6.0 (end of April)
+ - Adding support for more currently unsupported docker-compose keys.
+ - Initial support for building containers locally.
+
+### 0.7.0 (end of May)
+ - Basic support for docker-compose v3 format.
+ - Improved support for building containers locally.
```

Comment:
:+1: for this, i'm working on the support for building containers locally as well as adding the ability to push them too. It'll be a bit difficult / cumbersome, but we'll get there. Most like a lot more flags and parameters are going to be added.